microsoft-edge/devtools-guide-chromium/source-maps-monitor/source-maps-monitor-tool.md

```diff
@@ -25,6 +25,29 @@ To try this feature:
    ![The Source Maps Monitor tool](./source-maps-monitor-tool-images/source-maps-monitor-tool.png)
 
 
+<!-- ====================================================================== -->
+## Exclusions and inclusions
+
+The **Source Maps Monitor** tool shows which URL lookups have or haven't been attempted, as configured in DevTools **Settings > Symbol Server > Authorization mode = Azure Active Directory > Filter behavior = Exclusion List** or **Inclusion List**.  For example:
+
+1. In DevTools **Settings > Symbol Server**, in the **Authorization mode** dropdown list, select **Azure Active Directory**.
+
+1. Double-click the **microsoft** org to select it.
+
+1. In the **Filter behavior** dropdown list, select **Exclusion List**.
+
+1. In the **Filter behavior** text box, enter a domain with wildcard (`https://ryanseddon.com/*`) to exclude.
+
+1. Close DevTools **Settings**.
+
+As a result, the **Source Maps Monitor** tool indicates excluded source map URL lookup attempts:
+
+![Source Maps Monitor tool indicates source map URL lookups attempted](./source-maps-monitor-tool-images/source-maps-monitor-exclusions.png)
```

Comment:
user is likely to search Docs for "filter settings".  possibly:
* in TOC, add that as an Index entry.
* in article, add that string, for Ctrl+F Find in page.